R/driver.R

```diff
@@ -224,19 +224,28 @@ driver <- function(all_data = empty_data(),
     }
 
     if(!quiet) cat(nrow(unfound_inputs), "chunk data input(s) not accounted for\n")
-    csv_data <- load_csv_files(unfound_inputs$input, unfound_inputs$optional, quiet = TRUE)
-    all_data <- add_data(csv_data, all_data)
   }
 
   # Extract metadata from the input data; we'll add output metadata as we run
   metadata_info <- list()
-  if(return_data_map_only) {
-    if(!quiet) cat("Extracting metadata from inputs...\n")
-    metadata_info <- list(INPUT = tibbelize_outputs(all_data, chunk_name = "INPUT"))
-  }
 
   # Initialize some stuff before we start to run the chunks
-  chunks_to_run <- chunklist$name
+  if(!missing(stop_before) || !missing(stop_after)) {
+    run_chunks <- ifelse(missing(stop_before), stop_after, stop_before)
+    # calc min list
+    verts <- inner_join(bind_rows(chunkoutputs,
```

Comment:
See #830 - do we want to abstract this into a helper function, the start of a new data layer?